class_photo/__main__.py

```diff
@@ -1,4 +1,37 @@
+import os
+import sys
+from dotenv import load_dotenv
 from . import bot
+from . import face
+from . import collage
+
```

Comment:
Good function and variable naming practice. Consider adding descriptive comments around each function